preimages/preimages.asn

```diff
@@ -0,0 +1,42 @@
+-- Preimages STF test vectors schema
+
+PreimagesModule DEFINITIONS ::= BEGIN
+
+IMPORTS
+    TimeSlot, OpaqueHash, PreimagesExtrinsic
+        FROM JamTypes;
+
+
+State ::= SEQUENCE {
+    -- [..] Accounts.
+    accounts SEQUENCE OF AccountsMapEntry,
```

Comment:
I can't find definition of `AccountsMapEntry`

Reply:
Fixed by https://github.com/davxy/jam-test-vectors/pull/12